text/0110-z-pages.md

```diff
@@ -0,0 +1,35 @@
+# zPages: general direction (#110)
+
+Make zPages a standard OpenTelemetry component.
+
+## Motivation
+
+ZPages or self-introspection debug pages are in-process web pages that display collected data from the process they are attached to. They are used to provide in-process diagnostics without the need of any backend to examine traces or metrics. Various implementations of zPages are widely used in many environments. The standard extensible implementation of zPages in OpenTelemetry will benefit everybody.
```

Comment:
Is this `zPages` or `ZPages`?

Reply:
I think `zPages`. Just upper cased it in the beginning of a sentence. I changed the beginning of a sentence